draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -619,11 +623,221 @@ document MAY be accompanied by a respective claim suffixed with `#integrity`, in
+- `background_color`: A CSS color value for the background of the credential.
+  This property is OPTIONAL.
+- `text_color`: A CSS color value for the text of the credential. This property
+  is OPTIONAL.
+
+#### Logo Metadata {#logo-metadata}
+
+The `logo` property is an object containing information about the logo to be
+displayed for the type. The object contains the following properties:
+
+- `uri`: A URI pointing to the logo image. This property is REQUIRED.
+- `uri#integrity`: An "integrity metadata" string as described in
+  (#document-integrity). This property is OPTIONAL.
+- `alt_text`: A string containing alternative text for the logo image. This
+  property is OPTIONAL.
+
+### Rendering Method "svg_template" {#rendering-method-svg}
+
+The `svg_template` rendering method is intended for use in applications that
+support SVG rendering. The object MUST contain an array of objects containing
+information about the SVG templates available for the type. Each object contains
+the following properties:
+
+- `uri`: A URI pointing to the SVG template. This property is REQUIRED.
+- `uri#integrity`: An "integrity metadata" string as described in
+  (#document-integrity). This property is OPTIONAL.
+- `properties`: An object containing properties for the SVG template, as
+  described in (#svg-template-properties). This property is REQUIRED if more than
+  one SVG template is present, otherwise it is OPTIONAL.
+
+#### SVG Template Properties {#svg-template-properties}
+
+The `properties` property is an object containing properties for the SVG
+template. Consuming applications MUST use these properties to find the best SVG
+template available for display to the user based on the display properties
+(landscape/portrait) and user preferences (color scheme, contrast). The object
+MUST contain at least one of the following properties:
+
+- `orientation`: The orientation for which the SVG template is optimized, with
+  valid values being `portrait` and `landscape`. This property is OPTIONAL.
+- `color_scheme`: The color scheme for which the SVG template is optimized, with
+  valid values being `light` and `dark`. This property is OPTIONAL.
+- `contrast`: The contrast for which the SVG template is optimized, with valid
+  values being `normal` and `high`. This property is OPTIONAL.
+
+# Claim Metadata {#claim-metadata}
+
+The `claims` property is an array of objects containing information about
+particular claims for displaying and validating the claims.
+
+The array MAY contain an object for each claim that is supported by the type.
+Each object contains the following properties:
+
+- `path`: An array indicating the claim or claims that are being addressed, as
+  described below. This property is REQUIRED.
+- `display`: An object containing display information for the claim, as
+  described in (#claim-display-metadata). This property is OPTIONAL.
+- `verification`: A string indicating how the claim is verified, as described in
```

Comment:
to me, the phrasing "how the claim is verified" kinda suggests that it's something the verifier needs to be doing when verifying/validating the credential. I think ""how the claim was verified" better conveys what I think you're intent is.

```suggestion
- `verification`: A string indicating how the claim was verified, as described in
```